Fix/a11y with time tag for streams

xewl opened the pull request:
Following [yesterday's stream](https://www.youtube.com/watch?v=Xr0r2Ft_ROQ)
Let's provide a readable string (ISO8601, thus it includes the timezone) to the datetime attribute of the time tag.

- Added this to the preview card on top of the homepage
- Added to the local-time element, adjusted from `span` to `time` tag

Note:
- Seems like my vscode has re-sorted the tailwind classes, not sure if that's a problem.
- Some tests seem to fail, unrelated to this change. Also, not entirely sure if you want actual tests for these?
- I'm getting new versions of livewire files in public/ - push along?

christophrumpel commented:
Hey, thanks a lot for the PR 👍

Which tests are failing? At least the ones from Github Actions seem to be fine here.

Waht new files do you get from Livewire? Updated assets from Livewire?

xewl commented:
• When I run artisan test, I get ~10 of them failing with a lot of them spouting code around :D
eg.
```
   FAIL  Tests\Feature\Http\Livewire\StreamListArchiveTest
  ⨯ it only shows streams by selected streamer
  ⨯ it shows streamers as dropdown options
  
...


   FAIL  Tests\Feature\PageArchiveTest
  ⨯ it shows only finished streams
  ✓ it orders streams from latest to oldest
  ⨯ it does not show deleted streams
  ✓ it shows duration of stream if given
  ⨯ it searches for streams on title
  ⨯ it searches for streams on channel title
  ⨯ it searches for streams by specific streamer
  ⨯ it searches for streams by specific streamer and search term
```


• Yep, not sure if these are supposed to be .gitignore'd? I just restored them so they aren't in this PR.
```sh
        modified:   public/vendor/livewire/livewire.js
        modified:   public/vendor/livewire/livewire.js.map
        modified:   public/vendor/livewire/manifest.json
```

christophrumpel commented:
KK thanks for clarification. Let's merge this without the livewire files. I will take a look at them separately.
Thanks for the PR @xewl
